src/main/resources/data/pokecube_mobs/database/pokemobs/pokemobs_spawns.json

```diff
@@ -1516,6 +1516,14 @@
               "rate": "0.01",
               "max": "1"
             }
+          },
+		  {
+            "values": {
+              "structures": "pokecube_legends:lugia_tower",
+              "rate": "0.05",
+			  "min": "1",
```

Comment:
formatting would be fixed in you ran a json cleaner on here, I sent one in the discord a while ago, it is now pinned in the messages